daft/expressions/expressions.py

```diff
@@ -1186,6 +1186,7 @@ def minhash(
         num_hashes: int,
         ngram_size: int,
         seed: int = 1,
+        hash_function: native.HashFunctionKind = native.HashFunctionKind.MurmurHash3,
```

Comment:
Instead of an enum I think it's more preferred by end users to use a set of string literals instead.

Reply:
@samster25 we are using other enums throughout the code

![image](https://github.com/user-attachments/assets/6c9a3c6e-30c0-4006-bddb-ff75b466ff27)

There are differing opinions in the team on whether enums or string literals should be preferred (not talking about myself here). I think we should all get on the same page so our code can use consistent styling.

Reply:
Not sure who said that but those are should be internal and not exposed to the user. See `Join` in the dataframe:
https://github.com/Eventual-Inc/Daft/blob/main/daft/dataframe/dataframe.py#L1689C9-L1689C17

We convert to the enum here:
https://github.com/Eventual-Inc/Daft/blob/5b450fbd8af001c34eb9ee1a4724e497fa136e21/daft/dataframe/dataframe.py#L1783